### PR TITLE
Fix Extractor Import Path

### PR DESCRIPTION
**Pull request by pankajastro**
Fix extractor import path in docker-composer.yaml

### CODE REVIEW

File: dev/docker-compose.yaml [hunk 1]
```diff
@@ -23,8 +23,8 @@ x-airflow-common:
     AIRFLOW__LINEAGE__BACKEND: openlineage.lineage_backend.OpenLineageBackend
     OPENLINEAGE_URL: http://host.docker.internal:5050/
     OPENLINEAGE_EXTRACTORS: >-
-      astronomer.providers.google.cloud.extractors.bigquery_async_extractor.BigQueryAsyncExtractor
-      astronomer.providers.snowflake.extractors.snowflake_async_extractor.SnowflakeAsyncExtractor
+      astronomer.providers.google.cloud.extractors.bigquery.BigQueryAsyncExtractor
+      astronomer.providers.snowflake.extractors.snowflake.SnowflakeAsyncExtractor
```
Comment (pankajkoti) [inline dev/docker-compose.yaml:27]:
```suggestion
      astronomer.providers.google.cloud.extractors.bigquery.BigQueryAsyncExtractor;
      astronomer.providers.snowflake.extractors.snowflake.SnowflakeAsyncExtractor
```

Comment (pankajkoti) [reply dev/docker-compose.yaml:27]: Reference: https://github.com/OpenLineage/OpenLineage/tree/main/integration/airflow#custom-extractors

**Review by pankajkoti — APPROVED** [review]

**Comment by bharanidharan14** [comment]
can you add proper description to the PR

**Comment by bharanidharan14** [comment]
Is it ready for review ?

**Comment by pankajastro** [comment]
> @pankajastro Is it ready for review ?

yes

**Pull request merged**